packages/hooks/src/useLocalStorageState/index.ts

```diff
@@ -10,7 +10,7 @@ function useLocalStorageState<T>(
 ): [T, (value?: T | ((previousState: T) => T)) => void];
 
 function useLocalStorageState<T>(key: string, defaultValue?: T | (() => T)) {
-  return useStorageState(localStorage, key, defaultValue);
+  return useStorageState(() => localStorage, key, defaultValue);
```

Comment:
这里直接传 `null` 是不是就可以了？

```suggestion
  return useStorageState(window?.localStorage, key, defaultValue);
```

Reply:
当然更优雅的办法是传一个 `FakeStorage`，类似于 jest 的 mock，这样对底层的 `useStorageState` 来说，就无需感知是不是浏览器环境了

Reply:
![image](https://user-images.githubusercontent.com/21291247/93560806-8856ad00-f9b5-11ea-873e-fa5af72b502d.png)
这样写 也会报错吧

Reply:
然后为了在服务端不报错， 弄个 FakeStorage 感觉没啥必要吧

Reply:
嗯嗯OK，现在这样是可以work的，后面我再研究一下看能不能精简一下逻辑～